### PR TITLE
Pulling changes from vimaec/ara3d-dev fork

### PR DESCRIPTION
**Pull request by mavimaec**
Pulling changes from vimaec/ara3d-dev fork (branch: develop)

### CODE REVIEW

File: dotnet/Utilities/Util.cs
Comment (cdiggins) [inline dotnet/Utilities/Util.cs]: What happened here? @FrozenKiwi @mavimaec  do either of you know?

File: dotnet/UnityBridge/UnityConverters.cs
Comment (cdiggins) [inline dotnet/UnityBridge/UnityConverters.cs]: So we don't need this anymore, this coupling shouldn't be here.

**Review by cdiggins — CHANGES_REQUESTED** [review]

File: dotnet/Utilities/Util.cs
Comment (mavimaec) [reply dotnet/Utilities/Util.cs]: Seems like a change @FrozenKiwi made to compile to the Magic Leap 13 days ago.

Comment (FrozenKiwi) [reply dotnet/Utilities/Util.cs]: Yeah, I commented it out because (I think) DataTable wasn't available

This was one of those "ToDo's" that meant to come back to, but lost track of.  I think we might need to move this out of this project somewhere that isn't consumed by Unity

File: dotnet/UnityBridge/UnityConverters.cs
Comment (FrozenKiwi) [inline dotnet/UnityBridge/UnityConverters.cs]: We shouldn't have references to Revit specifically in Ara3D.

This could be renamed to something like "Ara3DToUnity" or something similar

Comment (FrozenKiwi) [reply dotnet/UnityBridge/UnityConverters.cs]: Also, it probably shouldn't be "To Unity".  Instead "YupRH" coordinate system (or whatever Unity is)

File: dotnet/UnityBridge/UnityConverters.cs
Comment (mavimaec) [reply dotnet/UnityBridge/UnityConverters.cs]: Correct. This dependency will be removed when the UnityLib branch is merged into `vimaec/ara3d-dev > develop`. This pull request will be on hold until that happens.

**Comment by mavimaec** [comment]
As per my comment above, this Pull Request on hold until the UnityLib project is merged into `vimaec-platform > develop` and all incorrect Revit dependencies are removed.

File: dotnet/UnityBridge/UnityConverters.cs
Comment (mavimaec) [reply dotnet/UnityBridge/UnityConverters.cs]: The Ara3D.Revit.DataModel dependency has been removed.

File: dotnet/UnityBridge/UnityConverters.cs
Comment (mavimaec) [reply dotnet/UnityBridge/UnityConverters.cs]: Renamed "ToUnityCoords"

File: dotnet/Utilities/Util.cs
Comment (mavimaec) [reply dotnet/Utilities/Util.cs]: I reinstated this code in the latest change. Should be good now.

**Comment by mavimaec** [comment]
@cdiggins @FrozenKiwi the latest commit should now have all the concerns addressed.

**Review by cdiggins — APPROVED** [review]
Looks good, thanks!

**Pull request merged**
